docs/SETUP.md

```diff
@@ -57,25 +57,15 @@ After initializing a Snapfu project, it will contain a template ready for you to
 npm run dev
 ```
 
-The local server will run at [https://localhost:3333](https://localhost:3333) and serves the contents of the `/public` directory
+The local server will run at [https://localhost:3333](https://localhost:3333) and serves the contents of the `/public` directory as well as the project bundle files. The development bundle files are be served from [https://localhost:3333/bundle.js](https://localhost:3333/bundle.js).
 
-The development bundle will be served from [https://localhost:3333/dist/bundle.js](https://localhost:3333/dist/bundle.js)
+Note: The local server uses a self-signed certificate for HTTPS, this will require adding a browser security exception for allowing the loading of these resources. There is a Chrome browser flag that can be set to prevent adding an exception: chrome://flags/#allow-insecure-localhost
 
-There are two ways we can develop and integration using the bundle:
-
-### Snapfu extension 
-
-[Snapfu extension](https://github.com/searchspring/snapfu-extension-chrome)
-
-Visit the website you are developing for. Click the extension to enable it and set the mode to `local`, then press `Save`.
-
-The page will reload and will load the local development bundle [https://localhost:3333/dist/bundle.js](https://localhost:3333/dist/bundle.js) onto the current website.
-
-The page will automatically reload upon saving any code modifications while `npm run dev` is running.
+Using Snapfu, there are two common ways to develop a project, using a local mockup file and leveraging the Snapfu Chrome extension.
 
 ### Local mockup
 
-Alternatively, you can create a local mockup `index.html` file within the `/public` directory that includes the bundle linked. The mockup can then be viewed at [https://localhost:3333](https://localhost:3333)
+Creation of a local mockup `index.html` file within the `/public` directory allows for the quickest development of a Snap integration. Mockup files can be created by copying the source files of particular pages that will contain the Snap integration - typically the search and category pages. These pages will need to include a link to the local bundle file (`bundle.js`) in order for the Snap bundle to run. A minimal mockup file (shown below) contains only the necessary script and target elements to get components rendered on the page. More complete mockups would normally contain DOM structure and links to website resources that would be present on the live site (CSS, etc...).
```

Comment:
nitpick - technically it doesn't have to be an index.html file.. we often see mockup.html. Looks like you call it mockup.html in the change below.

Reply:
Yea, this could be a bit more generic I suppose. It is meant only to be an example.